src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,77 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { Loader } from '../Loader';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions } from '../../features/currentTodo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
 
 export const TodoModal: React.FC = () => {
+  const [currentUser, setCurrentUser] = useState<User | null>(null);
+  const [isProcessing, setIsProcesing] = useState(false);
```

Comment:
There is a typo in the function name 'setIsProcesing'. It should be 'setIsProcessing'. Typos can lead to bugs and confusion in the codebase.